ddtrace/_worker.py

```diff
@@ -50,6 +51,7 @@ def start(self):
         """Start the periodic worker."""
         _LOG.debug('Starting %s thread', self._thread.name)
         self._thread.start()
+        self.started = True
```

Comment:
Maybe make this a `property` to avoid the user changing its value while the worker is running?